public/index.html

```diff
@@ -92,13 +92,14 @@
     <!-- Different font key for development and production -->
     <!-- Font will not work in development and will cause a 403, because the only domains allowed are *.gordon.edu -->
     <!-- Prod font key @PROD -->
-    <link
+    <!-- <link
       rel="stylesheet"
       type="text/css"
       href="https://cloud.typography.com/7763712/7294392/css/fonts.css"
-    />
+    /> -->
 
     <!-- Dev font key @TRAIN -->
+    <!-- Does not currently work, returns 403 -->
```

Comment:
This is actually expected to fail in Train, since we only have the rights to use the font in one website (and prod and train are technically different sites).